.github/actions/build_vgg_resnet_action/action.yml

```diff
@@ -0,0 +1,75 @@
+name: Setup and build networks; VGG16, Resnet50
+description: Build and run VV16 and Resnet50 with oneAPI base toolkit and oneAPI-Construction-Kit
+
+inputs:
+  workspace:
+    description: 'Main workflow workspace'
+    default: ${{ github.workspace }}
+
+runs:
+  # We don't want a new docker just a list of steps, so mark as composite
+  using: "composite"
+  steps:
+    - name: setup python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+
+    - name: Install Python libraries and system dependencies
+      shell: bash
+      run: |
+        pip install requests numpy h5py Pillow
+        sudo add-apt-repository universe
+        sudo apt-get update
+        sudo apt-get install -y libblas-dev libopenblas64-dev libopenblas-dev libpthread-stubs0-dev libboost-all-dev
+
+    - name: Clone portDNN
+      shell: bash
+      run: git clone --recursive https://github.com/codeplaysoftware/portDNN.git
+
+    - name: Download Daily Release
+      shell: bash
+      run: |
+        wget "https://github.com/intel/llvm/releases/download/nightly-2023-12-18/sycl_linux.tar.gz"
+        mkdir linux_nightly_release
+        tar -xzf sycl_linux.tar.gz -C linux_nightly_release
+
+    - name: Set up environment and build networks
+      shell: bash
+      run: |
+        ls
+        export LD_LIBRARY_PATH=${{ inputs.workspace }}/build/lib:${{ inputs.workspace }}/linux_nightly_release/lib/libsycl.so:${{ inputs.workspace }}/linux_nightly_release/lib:$LD_LIBRARY_PATH
+        export CMAKE_CXX_COMPILER=${{ inputs.workspace }}/linux_nightly_release/bin/clang++
+        export CMAKE_C_COMPILER=${{ inputs.workspace }}/linux_nightly_release/bin/clang
+        export CA_HAL_DEBUG=1
+        export CA_PROFILE_LEVEL=3
+        export ONEAPI_DEVICE_SELECTOR=opencl:acc
+        export OCL_ICD_FILENAMES=${{ inputs.workspace }}/build/lib/libCL.so
+        # To whitelist oneapi-construction-kit for the official Intel oneAPI basetoolkit,
+        # SYCL_CONFIG_FILE_NAME needs to be overridden
```

Comment:
I do not understand what this comment means. Can you explain a bit more?

Reply:
>    As the release has a whitelist of devices, it filters out RefSi. To override it, as a temporary solution we can point `SYCL_CONFIG_FILE_NAME` to ``. This way it doesn't set the default `sycl.conf`.